doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -1,13 +1,22 @@
 [id='test-designer-run-test-proc']
-= Running a test scenario
+= Running the test scenarios
 
 After creating a test scenario template and defining the test scenarios, you can run the tests to validate your business rules and data.
 
 .Procedure
-. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
-. The *Test Report* panel pops out from the right of the designer and displays the test result overview and the scenario status.
-. After the tests are executed, click *View Alerts* to open the *Alerts* panel at the bottom of the designer for the error messages.
-. In case the tests fails, review the error messages. Messages with a red cross indicate test failures where as those with a green check mark indicate successful tests.
+. To run defined test scenarios, do one of the following tasks:
+* To execute all the available test scenarios in your project inside multiple assets, in the upper-right corner of your project page, click *Test*.
 +
-In the designer, erroneous rows or cells are highlighted in red.
-. Make the necessary changes and run the test again till the scenario passes.
+.Run all the test scenarios from the project view
+image:AuthoringAssets/test-scenarios-run-all-tests-button.png[Run all the test scenarios from the project view]
++
+* To execute all the available test scenarios defined in a single `.scesim` file, at the top of the Test Scenario designer, click the *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
+* To run a single test scenario defined in a single `.scesim` file, right-click the row of the test scenario you want to run and select *Run scenario*.
+. The *Test Report* panel displays the overview of the tests and the scenario status.
```

Comment:
Suggestion - as this is not a step, change this step to a paragraph indented to the main step.